src/marlin_stubs/pause/M701_M702.cpp

```diff
@@ -51,14 +51,16 @@
 #define DO_NOT_RESTORE_Z_AXIS
 static const constexpr uint8_t Z_AXIS_LOAD_POS = 40;
 static const constexpr uint8_t Z_AXIS_UNLOAD_POS = 20;
+static const constexpr uint8_t X_AXIS_LOAD_POS = X_MAX_POS - X_MAX_POS / 4;
```

Comment:
use `size_t`, `uint8_t` is nearly too small

Reply:
if it should be same as `NOZZLE_PARK_POINT_M600[0]` make it so (do not copy that calculation)
use something like
```
static constexpr size_t park_point[] = NOZZLE_PARK_POINT_M600;
static constexpr size_t  X_AXIS_LOAD_POS = park_point[0];
```